index.html

```diff
@@ -5,11 +5,56 @@
     <title>Task2-1</title>
   </head>
   <body>
-
-
+      <h1 class="main-title">FizzBuzz問題</h1>
+      <p>
+        FizzNum：
+        <input type="text" name="number" id="fizz" placeholder="整数値を入れてください">
+      </p>
+      <p>
+          BuzzNum：
+        <input type="text" name="number" id="buzz" placeholder="整数値を入れてください">
+      </p>
+      <button type="button" name="name" value="value" id="btn">実行</button>
+      <div id="output">
+        <p>[出力]</p>
+      </div>
     <script>
       'use strict';
-      
+      const btn = document.getElementById('btn');
+      btn.addEventListener('click', () => {
+        const FizzNumber = Number(document.getElementById('fizz').value);
+        const BuzzNumber = Number(document.getElementById('buzz').value);
```

Comment:
javascript は変数を`camelCase` で記述することがほとんどなので
以下の通りにして欲しいです。

```suggestion
        const fizzNumber = Number(document.getElementById('fizz').value);
        const buzzNumber = Number(document.getElementById('buzz').value);
```